server.js

```diff
@@ -18,8 +19,114 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(listEndpoints(app));
 });
+////// Tuesday ////
+
+const { Schema } = mongoose;
+const HappyThoughtsSchema = new Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5, 
+    maxlength: 140
+  },
+  hearts: {
+    type: Number,
+    default: 0 
+
+  },
+  createdAt: {
+    type: Date,
+    default: () => new Date()
+  },
+}); 
+
+const HappyThoughts = mongoose.model("HappyThoughts", HappyThoughtsSchema);
+
+app.get("/thoughts", async (req, res) => {
+  try {
+    const thoughts = await HappyThoughts.find().sort({ createdAt: -1 }).limit(20);
+    res.status(200).json({
+      success: true,
+      response: thoughts,
+      message: "Thoughts retrieved successfully"
+    });
+  } catch(e) {
+    res.status(400).json({
+      success: false,
+      response: e,
+      message: "did not successfully"
```

Comment:
Agree with Johanna on this one. A suggestion might be something like:

"Failed to retrieve thoughts"